docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -1,49 +1,58 @@
 [[run-digger-installer]]
-== Run Digger Installer
+=== Run Digger Installer
 
-.To run the Digger installer, follow these steps:
 
-. Clone the https://github.com/aerogear/digger-installer[Digger Installer GitHub repo^]
+. Clone the https://github.com/aerogear/aerogear-digger-installer[Digger Installer GitHub repo^]
+
+. Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure. https://github.com/aerogear/aerogear-digger-installer/blob/master/inventory-sample[Sample Ansible Inventory file^].
 
-. Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure
 +
-NOTE: For a full list of variables that can be configured via the inventory file see the link:#installer-variables[Installer Variables] section.
+NOTE: For a full list of variables that can be configured via the inventory file refer to <<installer-variables>>
 
-. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal.:
+. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal:
++
+NOTE: Review https://github.com/aerogear/aerogear-digger-installer/blob/master/sample-build-playbook.yml[Digger Playbook^].
```

Comment:
I don't think we need to ask the user to review the playbook. We don't expect them to change it. There may be some other motivation for including as a note here..

Reply:
@laurafitzgerald could we remove the word "sample" from the playbook name?
can i raise a jira re this?

Reply:
NOTE: Refer to https://github.com/aerogear/aerogear-digger-installer/blob/master/sample-build-playbook.yml[Digger Playbook^] to see the roles executed during installation and their associated tags.

Reply:
resolved